### ISSUE TITLE
Page footer sits on top of everything else

### ISSUE THREAD

**Issue by irfansharif** [cockroachdb/docs#6773]
Consider https://www.cockroachlabs.com/docs/stable/information-schema.html for e.g., and scroll all the way to the bottom. The page footer is the top most view, hiding the columns on each side. This is not a common pattern elsewhere I believe.

<img width="1254" alt="image" src="https://user-images.githubusercontent.com/10536690/76101011-b1749a80-5f9b-11ea-8230-5b82440b42ac.png">

**Comment by jseldess** [comment]
@anagio, I thought I could figure this out quickly. Wrong. I'm confident you can, though, when there's time.

**Comment by luciaaaaaaa** [comment]
Hi, following up on this as part of the redesign. 

The footer only hides the side columns if scrolling in the main section, if the user needs to see the side panels they can scroll over those areas. @irfansharif do you have more details around the issue you experienced?

**Comment by irfansharif** [comment]
That's all the detail I had, unless you were asking for something specific? My reason for filing was that intuitively I imagine the footer to be "static" and "attached" to the end of the webpage, not a separate component altogether. So when the columns fold underneath the footer, it breaks that intuition. It now seems that the "static attachment" only applies to the main section, and not the columns on the side, which isn't what I was accustomed to.

tl;dr when I scroll down, I'm only looking to paginate down, so I expect things to fold out of view at the top panel of my browser, not at the bottom. I'm probably not doing the best job explaining it.

**Comment by jseldess** [comment]
@luciaaaaaaa, the footer here behaves like Irfan is requesting: https://learn.hashicorp.com/vault/getting-started/install

**Comment by jseldess** [comment]
^ I think those docs are really nicely designed, btw.

**Comment by luciaaaaaaa** [comment]
Closing in favor of footer obscures right nav #9357
